### PR TITLE
Getmethods

### PR DESCRIPTION
**Pull request by andrejro2**
Added method for getting payment metods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Requirements
 Installation
 ------------
 ```sh
-$ composer require renat-magadiev/comgate-client
+$ composer require andrejro2/comgate-client
```
Comment (renat-magadiev) [inline README.md]: @andrejro2 revert please

File: composer.json [hunk 2]
```diff
@@ -1,17 +1,21 @@
 {
-  "name": "renat-magadiev/comgate-client",
+  "name": "andrejro2/comgate-client",
```
Comment (renat-magadiev) [inline composer.json]: @andrejro2  revert please

File: src/Request/CreatePayment.php [hunk 3]
```diff
@@ -7,595 +7,578 @@
...
-     */
-    public function getEndPoint()
-    {
-        return 'create';
-    }
+  /**
+   * @return bool
+   */
+  public function needParse() {
+    return TRUE;
```
Comment (renat-magadiev) [inline src/Request/CreatePayment.php:566]: @andrejro2 it present BC break, make property with default TRUE value

File: src/Request/GetMethods.php [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+   */
+  public function isPost(): bool {
+    return FALSE;
+  }
+
+  /**
+   * @return bool
+   */
+  public function needParse() {
+    return FALSE;
```
Comment (renat-magadiev) [inline src/Request/GetMethods.php:70]: @andrejro2 same as create payment

File: src/Response/CreatePaymentResponse.php [hunk 5]
```diff
@@ -49,10 +47,6 @@ public function __construct(array $rawData)
             throw new InvalidArgumentException('Missing "message" in response');
         }
 
-        if (!$this->isOk()) {
```
Comment (renat-magadiev) [inline src/Response/CreatePaymentResponse.php:52]: @andrejro2 what's reason?

**Review by renat-magadiev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andrejro2) [reply README.md]: @renat-magadiev reverted

File: composer.json (hunk 2 above)
Comment (andrejro2) [reply composer.json]: @renat-magadiev reverted

File: src/Response/CreatePaymentResponse.php (hunk 5 above)
Comment (andrejro2) [reply src/Response/CreatePaymentResponse.php:52]: @renat-magadiev I don't remember to be honest, It's an old times story:) Reverted this change

**Comment by renat-magadiev** [comment]
thanks for contribute @andrejro2

**Pull request merged**